packages/eslint-plugin/src/rules/nx-plugin-checks.ts

```diff
@@ -14,6 +14,7 @@ import * as path from 'path';
 import { createESLintRule } from '../utils/create-eslint-rule';
 import { readProjectGraph } from '../utils/project-graph-utils';
 import { valid } from 'semver';
+import { join } from 'path';
```

Comment:
Can we use `joinPathFragments` instead of `join` here?

Reply:
Will that not use unix-style paths instead of os-style paths? 
`registerTsProject` passes the path to `swc` or `ts` which will attempt to read from the file system, so it would need to be os-style paths, right?

Reply:
No, this is not part of a generator and we are referring to paths on the real disk. They need to be what the OS expects, and may not be on the C: drive on windows.